LEAF_Request_Portal/admin/templates/mod_form.tpl

```diff
@@ -1178,22 +1165,49 @@ function getForm(indicatorID, series) {
             event.preventDefault();
         }
     });
+    $('#disabled').keypress(function(e){
+        if((e.keyCode ? e.keyCode : e.which) === 13){
```

Comment:
This is still not correct.  Pull the conditional assignment out of the if statement.

Reply:
Went through and pulled the ternary out of these